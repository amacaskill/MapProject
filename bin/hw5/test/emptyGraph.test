CreateGraph graph1

ListNodes graph1