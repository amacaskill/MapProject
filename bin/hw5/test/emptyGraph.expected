created graph graph1

graph1 contains: